tests/data/test_particle.py

```diff
@@ -13,8 +13,6 @@ def test_parity():
     parity = Parity(+1)
     assert parity == +1
     assert int(parity) == +1
-    with pytest.raises(AttributeError):
-        parity.value = -1
```

Comment:
Why is this removed?

Reply:
Mypy already checks for read-only access. If we do want to test read-only access, we could add `# type: ignore` here.